extensions/vercast/package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "vercast",
   "title": "Vercel Deployments",
-  "description": "Quickly open and check the status of your Vercel deployments.",
+  "description": "View your Vercel deployments",
   "icon": "icon.png",
   "author": "matt",
   "license": "MPL-2.0",
```

Comment:
@gleich Could you please update the license to MIT here?

Reply:
Yeah, please also remove the license file. We have a LICENSE file already in the repo's root folder and require all extension to have MIT.

Reply:
ah ok, sounds good